packages/itwin/grouping-mapping-widget/README.md

```diff
@@ -72,7 +72,7 @@ new GroupingMappingProvider({
 
 After adding the grouping custom UI, it will be presented in the 'Add Group' drop down list.
 
-![Process Functional Grouping Menu Item](./assets/ProcessGroupingMenuItem.jpg)
+![Process Functional Grouping Menu Item](https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/grouping-mapping-widget/assets/ProcessGroupingMenuItem.jpg)

```

Comment:
Instead of providing absolute paths in the readme, you could try to change the `repository` object in package.json.

Change `repository.url` to `https://github.com/iTwin/viewer-components-react`
Add `repository.directory` to `packages/itwin/grouping-mapping-widget`

See documentation here: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#repository

Absolute paths are error prone, because if the repo is moved somewhere you will have to change all of the links everywhere. You would only need to change 1 link in package json with relative paths.

Reply:
Thank you for reviewing. Just FYI, this PR is still draft, will not be published soon

Reply:
Why won't it be published soon? There is a bug in this package (npmjs readme), and this PR fixes might fix it. Why not merge it?

Reply:
please review and let's merge it